unity/Assets/Scripts/AgentManager.cs

```diff
@@ -1538,7 +1551,7 @@ public class DynamicServerAction {
 
     public int sequenceId {
         get {
-            return (int)this.jObject["sequenceId"];
+            return this.GetValue("sequenceId", 0);
```

Comment:
This should be reverted.  We can't default the sequenceId as it should always originate from the Python side.  If the sequenceId is missing, then something is really wrong.  It would be better to just check to see if it exists and throw an error if it doesn't.

Reply:
Fixed as conversation we had